packages/docs/docs/using-masca.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 2
+---
+
+# Using Masca
```

Comment:
On this page, add sections for poylgon id and oidc:

## Polygon ID

Learn more on this page.

## OpenID for Verifiable Credentials (OID4VC)

Learn more on this page.